e2e/browser/solid-client-authn-browser/test/fixtures.ts

```diff
@@ -275,6 +276,7 @@ export interface ISeedPodResponse {
 
 export async function seedPod(
   setupEnvironment: TestingEnvironmentNode,
+  enableRefresh?: boolean,
```

Comment:
The changes here are all about allowing a test to use the refresh flow but by default it is still not enabled